go.mod

```diff
@@ -1,13 +1,13 @@
 module volcano.sh/apis
 
-go 1.21
+go 1.22
```

Comment:
Does v1.30 need go version 1.22?

Reply:
I have refer to: https://github.com/kubernetes/kubernetes/blob/release-1.30/go.mod 
So I think that k8s v1.30 using go version 1.22. Is it right?